_maps/map_files/MetaStation/MetaStation.dmm

```diff
@@ -55195,8 +55195,7 @@
 	heat_proof = 1;
 	name = "Supermatter Chamber"
 	},
-/obj/effect/mapping_helpers/airlock/access/any/engineering/atmos,
-/obj/effect/mapping_helpers/airlock/access/any/engineering/general,
+/obj/effect/mapping_helpers/airlock/access/all/engineering/atmos,
```

Comment:
This change needs undoing, we want the two `any` helpers here, not two `all` helpers.